### ISSUE TITLE
Autosave after x minutes

### ISSUE THREAD

**Issue by alexhammerschmied** [Webmecanik/plugin-grapesjs-builder#34]
It would be great to introduce an autosave function after x minutes or some kind of reminder popup for saving.

**Comment by TornMarketing** [comment]
If we did that, I would be rather keen for version control in a similar way that Wordpress manages version control with auto save

**Comment by alexhammerschmied** [comment]
Yes that's true. Autosave is only possible if we make revisions accessible somehow. 
So for the beginning I would suggest we create a simple popup message telling you after 10 minutes to save.
